client/src/components/App.jsx

```diff
@@ -27,6 +27,9 @@ class App extends React.Component {
   render () {
     return (
       <div>
+        <div>
+          <p> <i class="fas fa-comment-alt"></i> {this.state.comments.length} comments </p>
```

Comment:
Style guide requests that empty tags are self closing, like
<i class="..." />

Reply:
Okie doke.